src/main/kotlin/com/github/seliba/devcordbot/core/autohelp/AutoHelp.kt

```diff
@@ -30,84 +28,87 @@ import com.github.seliba.devcordbot.dsl.sendMessage
 import com.github.seliba.devcordbot.event.EventSubscriber
 import com.github.seliba.devcordbot.util.HastebinUtil
 import com.github.seliba.devcordbot.util.await
-import com.github.seliba.devcordbot.util.executeAsync
+import kotlinx.coroutines.GlobalScope
+import kotlinx.coroutines.asCoroutineDispatcher
 import kotlinx.coroutines.future.await
-import kotlinx.coroutines.runBlocking
+import kotlinx.coroutines.future.future
 import net.dv8tion.jda.api.entities.Message
 import net.dv8tion.jda.api.events.message.guild.GuildMessageReceivedEvent
-import okhttp3.Request
 import org.jetbrains.exposed.sql.transactions.transaction
 import java.io.BufferedReader
 import java.io.InputStreamReader
 import java.util.concurrent.CompletableFuture
+import java.util.concurrent.Executors
 
-private const val MAX_LINES = 15
-
-/**
- * Automatic analzyer for common pitfalls.
- */
-class CommonPitfallListener(
+class AutoHelp(
     private val bot: DevCordBot,
     private val whitelist: List<String>,
     private val blacklist: List<String>,
     knownLanguages: List<String>
 ) {
-    private val languageGuesser = Highlighter(UselessRendererFactoryThing())
-    private val knownLanguages = knownLanguages.toTypedArray()
 
-    /**
-     * Listens for new messages.
-     */
+    private val gussr = LanguageGusser(knownLanguages)
```

Comment:
You may fix the typo

Reply:
https://www.geoguessr.com/ triggered me too much with this name

Reply:
This is obviously not resolved.